src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.trim() !== '');
+
+  const stylesObject = styles.reduce((acc, style) => {
+    const [property, value] = style.split(':', 2)
+      .map((item) => item.trim());
+
+    if (property.length > 0 && value.length > 0) {
+      acc[property] = value;
+    }
+
+    return acc;
```

Comment:
The use of `reduce` here is appropriate for transforming an array into an object. However, be cautious with the use of magic numbers like the '2' in the `split` method. It's not immediately clear why '2' is used, so a comment explaining its purpose would be beneficial for readability and maintainability.